microsoft-edge/devtools/experimental-features/index.md

```diff
@@ -185,8 +185,28 @@ See also:
 * [Record heap snapshots using the Memory tool ("Heap snapshot" profiling type)](../memory-problems/heap-snapshots.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 136.
-* This checkbox is present in Microsoft Edge Stable 135.
+* This checkbox is present in Microsoft Edge Canary 140.
+* This checkbox is present in Microsoft Edge Stable 138.
+
+
+<!-- ====================================================================== -->
+## Enable vertical drawer configuration
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 139_
+
+This checkbox has no effect.
+
+* In Microsoft Edge DevTools, the **Quick View** panel's toolbar has a **Dock Quick View to the right** button, or a **Dock Quick View to the bottom** button.
+
+* In Google Chrome DevTools, this checkbox adds a **Toggle drawer orientation** button on the **Drawer** toolbar.
```

Comment:
I don't think we should document Chrome DevTools.
It's ok to mention the checkbox because the checkbox exists. But we shouldn't go beyond documenting what it does in Edge.

```suggestion
This checkbox has no effect. You can toggle the **Quick View** orientation by using the **Dock Quick View to the right** button.
```